tests/queries/0_stateless/02864_statistic_uniq.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS t1;
```

Comment:
This test looks similar to 02864_statistic_operate.sql. What if we combine it with the other file? That will reduce the risk to change stuff in one file but then to forget changing the other file.

Reply:
well, I want to avoid a bug test file, so I made different statistics type in a different file